src/SimpleSelect.js

```diff
@@ -18,6 +19,7 @@
         options);
       }),
       firstOptionIndexToHighlight: id,
+      listboxId: uuid(),
```

Comment:
adding a package just for a default prop value seems overkill to me.  it seems acceptable to me to just leave the default id null, and then if you don't want that to be null, you can pass in the listboxId prop.

Reply:
it's necessary for there to be an id, but that id doesn't need to be known to the outside world because It's just what links the control to the listbox that pops up.  It's not something that we would want to provide in our app (but since it's an id we also want it to be unique).

Reply:
I've updated this so that listboxId isn't something that can be passed in (it's always going to be generated).